go/stellar/stellar.go

```diff
@@ -64,7 +65,7 @@ func CreateWallet(ctx context.Context, g *libkb.GlobalContext) (created bool, er
 		g.Log.CErrorf(ctx, "We've just posted a bundle that's missing PrimaryAccount: %s", err)
 		return false, err
 	}
-	if err := remote.SetAccountDefaultCurrency(ctx, g, primary.AccountID, "USD"); err != nil {
+	if err := remote.SetAccountDefaultCurrency(ctx, g, primary.AccountID, DefaultCurrencySetting); err != nil {
```

Comment:
Let's remove this block.  

I don't think we should save this when wallets are created, let the user set it.  Confirmed with @malgorithms to remove it.

Reply:
It's purely technical, though, isn't it? it will still return the default "USD" for non-set accounts.

Also it was put there as preparation for when we can detect region/locale and set it accordingly - https://keybase.atlassian.net/browse/CORE-8071

Reply:
or do we want to support "outside-currency-less" accounts? both are fine by me, I just want to ensure we are not forgetting anything that we discussed prior

Reply:
Yes, but we want to know that the user didn't choose USD, we are just using that as a default if they have nothing set.

Reply:
That way we can change the default to EUR or whatever and it will work fine.